public/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>TimeStamp</title>
+    <link rel="stylesheet" href="/css/main.css" />
+  </head>
+  <body>
+    <div class="center"><h1>TimeStamp Microservice</h1></div>
+
+</br>
+</br>
+<div class="center">
+<h1>Example Usage:</h1>
+</div>
+</br>
+
+<div class="center">
+    <h3>Natural Language:</h3>
+</div>
+
+<div class="center">
+        <p class="highlight">http://time.jma.cloud:3000/feb%2025,%202017 </p>
+        <p></p>
```

Comment:
Is there a reason you have an empty <p>

Reply:
I forgot it was there, I have removed it